### DISCUSSION TITLE
Sign out of all devices on an account

### DISCUSSION THREAD

**Discussion by owen-dean** [monkeytypegame/monkeytype#1910] [Ideas]
I've been having people sign into my account, and I can't figure out who. I can't sign them out, because even if I change my password, they can still get in. I think this could be a useful feature for security.

**Comment by Miodec** [comment]
How do you know people have been signing into your account? I think if you change your password the auto login will not work.

**Reply by owen-dean** [reply]
All of my settings got changed. Theme, font colors, font size, etc.

**Reply by unrealapex** [reply]
Could you provide a copy of your settings JSON(open command line, type `Export settings JSON`, copy and paste)? Also are there multiple people who use Monkeytype on your computer?

**Reply by owen-dean** [reply]
[Untitled document.docx](https://github.com/Miodec/monkeytype/files/7252835/Untitled.document.docx)
Here's the settings, but I've changed them back from then. Also, I log into some of my friends' computers sometimes to show them my scores, but I make sure to log out everytime I'm done.

**Reply by unrealapex** [reply]
Does the config look like your friend's? One possible problem might be the settings stored in cookies that are in your friends' computers. Say for example, a friend of yours used the rgb theme, but you use the serika dark(the default). It might be possible that the previously set value for the theme might have been set for your account too when you logged in. If that is the case, this would apply for all your settings.

**Reply by owen-dean** [reply]
that could be the problem, I'll check
